queries.sql

```diff
@@ -209,3 +209,128 @@ JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY C
  Sam Smith       |             1
 (5 rows)
 */
+
+-- Who was the last animal seen by William Tatcher?
+
+SELECT animals.name, MAX(visits.date_of_visit) FROM animals
+JOIN visits ON visits.animals_id = animals.id WHERE vets_id = 1 GROUP BY animals.name ORDER BY MAX(visits.date_of_visit) DESC;
```

Comment:
- [ ] This query is good however, it's returning all animals seen by William Tatcher but we're asked to get the last animal seen by William Tacher. You can add `LIMIT 1;` to the end of your query so that your query returns the last animal seen by Willam Tatcher.

Reply:
Hi Juliana, thank you for reviewing the code, I actually discovered `LIMIT` halfway through the project and I totally forgot to add it into that query. Thank you for pointing out 😃